### DISCUSSION TITLE
Latin XS

### DISCUSSION THREAD

**Discussion by colinmford** [koeberlin/Latin-Character-Sets#7] [Ideas]
I was thinking recently, I love these character sets, but I wonder if you've given thought to a character set smaller than Latin S, let's call it Latin XS. 

Often one might be making a proof of concept font, or a small font for a client, and one might want to have just a limited character set, akin to Latin-1. However, we all know that Latin-1 has many limitations, [like not fully supporting all of the languages is purports to support](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#Languages_with_incomplete_coverage).

It would be very interesting to see your take on a "minimal" character set that still does a good job with most "popular" languages (though of course, where does one draw the line?).

**Comment by koeberlin** [comment]
Thank you for your kind words, Colin! Initially I thought Latin S as the smallest would make more designers question their base character set. But indeed it might be handy to have something for the cases you mention. Would you be interested in developing it together?
